public/css/common.less

```diff
@@ -212,15 +212,29 @@ div.show-more {
   }
 }
 
-.status-bar {
+.footer {
   display: flex;
-  justify-content: space-between;
-  align-items: center;
-  padding: .25em 1em;
-  box-shadow: 0 -1px 0 0px @gray-lighter;
-  font-size: .857em;
-  z-index: 1000;
```

Comment:
Please re-add the `z-index`. There's no need to remove it.

Reply:
Though, you should add it to the footer instead since it's not only the status bar anymore which is part of it.